rust/ps-logger/README.md

```diff
@@ -0,0 +1,11 @@
+# Scylla tutorial
```

Comment:
Please add more context to the README
what is this example doing?
See example in Python, Java lessons